lib/tenios/api.rb

```diff
@@ -4,6 +4,6 @@
 
 module Tenios
   module API
-    autoload :Version, "tenios/api/version"
+    autoload :VERSION, "tenios/api/version"
```

Comment:
Would we also need to change the constant?

https://github.com/carwow/tenios-api-ruby/blob/506456213813b346cdf0778b9caadc70f209cb6d/lib/tenios/api/version.rb#L5

Reply:
No, they match now, they didn't before. Not sure how it worked before. 😅  But running the test locally, it was failing.